### PR TITLE
Transfer Escrow on loan repaid

### PR DESCRIPTION
**Pull request by passabilities**

### CODE REVIEW

File: contracts/base/Escrow.sol [hunk 1]
```diff
@@ -104,8 +156,24 @@ contract Escrow is EscrowInterface, TInitializable, Ownable, BaseUpgradeable, Ba
...
+        _tokenUpdated(loans.lendingToken());
     }
 
     /** Internal Functions */
+
+    function _valueOfIn(address baseAddress, address quoteAddress, uint256 baseAmount) internal view returns (uint256) {
+        uint8 baseDecimals = baseAddress == 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE ? 18 : ERC20Detailed(baseAddress).decimals();
+        PairAggregatorInterface aggregator = _getAggregatorFor(baseAddress, quoteAddress);
+        uint256 oneTokenPrice = uint256(aggregator.getLatestAnswer());
+        // TODO: better way with SafeMath?
```
Comment (salazarguille) [inline contracts/base/Escrow.sol:170]: Is this TODO fixed?

File: package.json [hunk 2]
```diff
@@ -42,7 +42,6 @@
     "leche": "^2.3.0",
     "lodash": "^4.17.19",
     "mocha": "^7.1.0",
-    "openzeppelin-test-helpers": "^0.5.1",
```
Comment (salazarguille) [inline package.json:45]: why it is removed?

**Review by salazarguille — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (passabilities) [reply package.json:45]: no longer needed since I'm testing for the event elsewhere

File: contracts/base/Escrow.sol (hunk 1 above)
Comment (passabilities) [reply contracts/base/Escrow.sol:170]: No I wasn't sure since there is no `mul` function in SafeMath.

**Review by salazarguille — APPROVED** [review]

**Pull request merged**